pyproject.toml

```diff
@@ -20,6 +20,7 @@ azure-mgmt-marketplaceordering = {version = "^1.1.0", optional = true}
 azure-mgmt-network = {version = "^19.3.0", optional = true}
 azure-mgmt-resource = {version = "^21.0.0", optional = true}
 azure-mgmt-serialconsole = {version = "^1.0.0", optional = true}
+azure-cli-core = {version = "^2.40.0", optional = true}
```

Comment:
Add an internal PR to update packages to prevent broken, when this PR is merged.